benefits/eligibility/templates/eligibility/includes/media-item--idcardcheck--start--senior.html

```diff
@@ -10,10 +10,8 @@
     <div class="media-body--details">
         <p>
             {% translate "eligibility.pages.start.senior.start_item.details" %}
-            <!-- Button to trigger modal -->
-            <button type="button" class="modal-trigger--link" data-bs-toggle="modal" data-bs-target="#identity-verification-help">
-                {% translate "eligibility.pages.start.senior.help.link_text" %}
-            </button>
+            {% translate "eligibility.pages.start.senior.help.link_text" as trigger_text %}
+            {% include "core/includes/modal-trigger-link.html" with id="identity-verification-help" classes="modal-trigger--link" text=trigger_text %}
```

Comment:
Interesting thing I learned: The above red code produces this tiny extra space between the end of the word
`verification` and the `(?)`


By passing in the trigger text into the includes instead, the mysterious extra space is removed.
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/206cd960-8cc4-4f30-85e9-11d1511bec53">
<img width="391" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/a554528c-e499-4089-836b-1c49d359b17e">

I think something about the new line might be adding a space or something. Not sure!